.gitignore

```diff
@@ -7,54 +7,29 @@ __pycache__/
 .vscode/
 .vs/
 .idea/
-CMakeLists.txt
-cmake-build-debug
-venv/
-.venv/
 
 # Project-specific ignores
 build/
 expected/
-notes/
-baserom/
-baseroms/*/segments/
 docs/doxygen/
-*.elf
-*.sra
 *.z64
 *.n64
 *.v64
-*.map
-*.dump
-out.txt
-*.ram
 
-# Tool artifacts
-tools/mipspro7.2_compiler/
-tools/overlayhelpers/batchdisasm/output/*
-tools/overlayhelpers/batchdisasm/output2/*
-tools/overlayhelpers/batchdisasm/mipsdisasm/*
+# Tools
+.venv/
 tools/disasm/output/*
```

Comment:
`tools/disasm/output/*` can probably be removed, it's not used by the current iteration of tools/disasm (according to blame it's from the initial commit)